crates/nu-engine/src/nu_variable.rs

```diff
@@ -213,6 +215,19 @@ impl LazyRecord for NuVariable {
                 val: self.engine_state.get_startup_time(),
                 span: self.span(),
             }),
+            "current-exe" => {
+                let exe = std::env::current_exe().map_err(|_| {
+                    err("Could not get current executable path")
+                        .expect_err("did not get err from err function")
+                })?;
```

Comment:
I generally like tihs idea. My only quesiton is, is this error handling what we want? I'm wondering if we should just do someting like
```rust
let exe = std::env::current_exe()?;
```
or if what you have here is better

Reply:
I just saw the `err` function being used above and also used that here. I'm not sure what the difference is between that and just using ?